weekly-updates/index.js

```diff
@@ -222,44 +284,54 @@ getReport = async () => {
   // Uncomment the following like to use static data instead
   // const zenhubData = await getZenhubInfoStatic();
   const zenhubData = await getZenhubInfo();
+  const closedIssuesData = await getClosedIssuesInfo();
   const workspaceData = zenhubData.data.viewer.searchWorkspaces.nodes[0];
+  
 
   // Figure out where in the sprint we currently are
   let currentSprintEndDate = workspaceData.activeSprint.endAt;
   let dateDiff = moment(currentSprintEndDate).diff(currentDate, "days");
 
   let plannedTasks;
   let closedTasks;
+  let spilledTasks;
 
   let enclosedIn = currentDate;
   if (enclosedIn.isoWeekday() === 1) {
     // Only on Mondays, we actually want to see the report for last week
     enclosedIn.subtract(3, "days");
   }
-  if (dateDiff < 3) {
+  if (dateDiff <= 3) {
     // This sprint is about to end, planned tasks will be taken from the next sprint
     plannedTasks = await getPlannedTasks(workspaceData.upcomingSprint);
```

Comment:
Over here should we consider the spillover as well since we do this every friday